installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -0,0 +1,92 @@
+[id="installing-ibm-z"]
+= Installing a cluster on IBM Z
+include::modules/common-attributes.adoc[]
+:context: installing-ibm-z
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+IBM Z infrastructure that you provision.
+
+[IMPORTANT]
+====
+While you might be able to follow this procedure to deploy a cluster on
+virtualized or cloud environments, you must be aware of additional
+considerations for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms]
+before you attempt to install an {product-title} cluster in such an environment.
+====
+
+.Prerequisites
+
+* Provision
+xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
```

Comment:
You need to update all of the heading information in the modules that you've included to state that they're used in this assembly.

Reply:
Headers updated